lib/pluginmanager/templates/codec-plugin/.ruby-version

```diff
@@ -0,0 +1 @@
+../../../../.ruby-version
```

Comment:
This is a symlink to the top-level logstash `.ruby-version`. This was added because without it, the newly generated project attempts to use your global/system ruby, which on mac defaults to MRI ruby. The error you get is:

```
$ bundle install

[!] There was an error while loading `logstash-core-plugin-api.gemspec`: The logstash-core-api need to be build on jruby. Bundler cannot continue.

 #  from /Users/seanstory/Desktop/Dev/logstash/logstash-core-plugin-api/logstash-core-plugin-api.gemspec:46
 #  -------------------------------------------
 #    else
 >      raise "The logstash-core-api need to be build on jruby"
 #    end
 #  -------------------------------------------
```